index.html

```diff
@@ -162,16 +175,18 @@ <h1>About me</h1>
     </div>
     <hr class="separating-line">
     <div class= "others">
-        <div class="language"><img src="images/Rectangle 55.png"> Language
-         </div>   
+        <div class="languages">
+        <div class="language-text"><img src="images/Rectangle 55.png"> Language
+        </div>
          <ul>
                 <li>Javascript</li>
                 <li>Ruby</li>
                 <li>Html</li>
                 <li>Css</li>
             </ul>
-        
-        <div class="frameworks"><img src="images/Rectangle 56.png" alt="rectang;e-56"> Frameworks
+            </div>
+        <div class="frameworks">
```

Comment:
- [ ] KIndly adjust the font size and font weight of your list of technologies an framework to match the Figma design 

Your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/215233947-e52041cf-098d-4561-911e-2d521beaf2a4.png)

Figma design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/215233987-c4237c80-6cbf-4753-8bcb-b3ed9843639a.png)